build-support/bin/release.py

```diff
@@ -389,6 +389,15 @@ def __init__(self) -> None:
             .stdout.decode()
             .strip()
         )
+        self._head_committer_date = (
+            subprocess.run(
+                ["git", "show", "--no-patch", "--format=%ct", self._head_sha],
```

Comment:
Could you do a human readable date? Maybe ~`YmdH`: e.g. `2023053024`

Reply:
You mean you can't read unix timestamps?!

... I can't either.

Will do.